pkg/ingester/label_names_and_values.go

```diff
@@ -161,14 +176,45 @@ func labelValuesCardinality(
 	return nil
 }
 
+func computeLabelValuesSeriesCount(
+	ctx context.Context,
+	lbName string,
+	lbValues []string,
+	matchers []*labels.Matcher,
+	idxReader tsdb.IndexReader,
+	postingsForMatchersFn func(tsdb.IndexPostingsReader, ...*labels.Matcher) (index.Postings, error),
+	countCh chan<- labelValueCountResult,
+) {
+	_ = concurrency.ForEachJob(ctx, len(lbValues), maxConcurrency, func(ctx context.Context, idx int) error {
```

Comment:
Can you add a comment explaining that errors are handled elsewhere (and hence why we're ignoring this return)?